docs/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,96 @@
+> Release Issue Template
+
+# go-ipfs X.Y.Z Release
+
+We're happy to announce go-ipfs X.Y.Z, bla bla...
+
+## 🗺 What's left for release
+
+<List of items with PRs and/or Issues to be considered for this release>
+
+## 🔦 Highlights
+
+< top highlights for this release notes >
+
+## Changelog
+
+< changelog generated by bin/mkreleaselog >
```

Comment:
This is going to be _really_ long: https://github.com/ipfs/go-ipfs/blob/master/CHANGELOG.md#changelog

Reply:
Seems that most of it is libp2p releated. Shouldn't that just appear on go-libp2p releases?

Reply:
It's about half libp2p. Once libp2p starts making releases with changelogs, we can just include a link to the libp2p release and leave it at that.

But that's still over 100 lines. Let's leave it for now but we may need to hide this behind a `<details>`.